designs/virtualization/4.5/Scheduling/Dedicated-resources/dedicated-resources.md

```diff
@@ -0,0 +1,23 @@
+---
+parent: virtualization
+version: 4.5
+---
+
+# Dedicated resources
+
+Enabling dedicated resources for a VM workload.
+
+![vm page](img/Dedicated-resources-0-0.jpg)
+'Dedicated resources' requirement is placed in the Scheduling section of the VM details tab. The default is 'No dedicated resources applied'
+
+![Dedicated resources modal](img/Dedicated-resources-1-0.jpg)
+This feature is available only for nodes with the label 'cpumanager=true'
+It is specifed within the modal.
+On landing on the modal, the nodes checker already checked for matching nodes.
+
+![Dedicated resources modal - no nodes found](img/Dedicated-resources-1-1.jpg)
```

Comment:
It could be more helpful to state 
"No matching nodes found for labels on this workload"
"Scheduling will not be possible"

Reply:
I changed the text to simply mention the policy name instead. WDYT?